DopplerFiles/Program.cs

```diff
@@ -12,6 +13,11 @@ public static void Main(string[] args)
 
         public static IHostBuilder CreateHostBuilder(string[] args) =>
             Host.CreateDefaultBuilder(args)
+                .ConfigureAppConfiguration(configHost =>
+                {
+                    configHost.AddJsonFile("/run/secrets/appsettings.Secret.json", true);
+                    configHost.AddKeyPerFile("/run/secrets", true);
```

Comment:
👍

And maybe this could be useful for you for local testing:

```
configurationBuilder.AddJsonFile("appsettings.Secret.json", true);
```

See https://github.com/FromDoppler/doppler-custom-domain/blob/882fe368e0dcfa645bf32375eaa7a8573c3f1ff4/DopplerCustomDomain/Program.cs#L21


It search for `appsettings.Secret.json` in the folder of your application.

Reply:
Ok, Thx